### PR TITLE
Change experimental fetch to node-fetch

### PR DESCRIPTION
**Pull request by Hrtkn**
Backend/github.js was using experimental fetch function that was introduced in Node 18. 
More on that here: https://nodejs.org/de/blog/announcements/v18-release-announce/
I changed it to use node-fetch, which does the same exact thing, but is not experimental.
https://www.npmjs.com/package/node-fetch

### CODE REVIEW

File: Backend/Dockerfile [hunk 1]
```diff
@@ -11,4 +11,4 @@ COPY . .
 
 EXPOSE 8080
 
-CMD ["node", "index.js"]
\ No newline at end of file
+CMD ["node", "--no-experimental-fetch", "index.js"];
```
Comment (xfkoko) [inline Backend/Dockerfile:14]: Is this change still needed?
Otherwise Ok, can merge after this is resolved.

Comment (xfkoko) [reply Backend/Dockerfile:14]: Ok, revet this and merge. Great work, thank!

**Review by irrelv — APPROVED** [review]
Looks great
